### DISCUSSION TITLE
你们这也太内卷了，market-1501都快被你们玩坏了

### DISCUSSION THREAD

**Discussion by CynicalHeart** [finger-monkey/Data-Augmentation#3] [General]
几天没上paper with code 就又有刷榜了，不过st-ReID这种加了相机拓扑预设的idea本来就很犯规（虽然在mtmc领域很香）。
看了下大致上就是用结合了自己的trick，用了哲东博士和罗浩老师的baseline，搞出plus版的reid。还有那个京东fast-reid，这玩意对我这种小白来说，用起来贼费劲。还是哲东博士的baseline最方便好用，但我练了各种参数都没把mAP刷上85，我好好奇你的超参呀（馋），希望早日完整开源，比心😘

**Comment by finger-monkey** [comment]
> 几天没上paper with code 就又有刷榜了，不过st-ReID这种加了相机拓扑预设的idea本来就很犯规（虽然在mtmc领域很香）。
> 看了下大致上就是用结合了自己的trick，用了哲东博士和罗浩老师的baseline，搞出plus版的reid。还有那个京东fast-reid，这玩意对我这种小白来说，用起来贼费劲。还是哲东博士的baseline最方便好用，但我练了各种参数都没把mAP刷上85，我好好奇你的超参呀（馋），希望早日完整开源，比心😘

抱歉现在才看到这条留言，已过了快半年......   
哲东博士的baseline确实是最友好的，后来看到许多顶会顶刊的代码都是基于它改造的。
现在是无监督在刷榜，看来内卷是第一生产力...
我的方法的超参数只有两个，在论文里有提到，剩下的都是直接用baseline里默认的。用在fasereid上时要把它代码里的autoAug自动数据增强关掉。

**Reply by finger-monkey** [reply]
论文里的方法验证起来十分容易，可以试一下。全局灰度替换的超参数设置推荐为0.05，局部灰度替换的超参数设置推荐为0.4.

**Reply by CynicalHeart** [reply]
实现起来的确容易，随即灰度擦除我就是直接按照随机黑块擦除改的，把原图灰度化后抠出来再覆盖原图，刚看到上个月就开源了，挺棒的！在哲东的那个baseline上Rank-1为92.4，提升了一点点。`fast-reid`和罗浩的baseline都用了模板，改起来就有点费劲。

**Reply by finger-monkey** [reply]
> 实现起来的确容易，随即灰度擦除我就是直接按照随机黑块擦除改的，把原图灰度化后抠出来再覆盖原图，刚看到上个月就开源了，挺棒的！在哲东的那个baseline上Rank-1为92.4，提升了一点点。`fast-reid`和罗浩的baseline都用了模板，改起来就有点费劲。

其实去年年底就开源了，后面有段时间因为特别的原因就删掉了。其实在什么trick都不使用的情况下提升是最明显的。加了随机擦除、学习率热身、平滑标签...之后，提升的空间就内卷了。
